### ISSUE TITLE
cannot pip

### ISSUE THREAD

**Issue by ZenithJoyH** [mklanac/pySolidWorks#1]
when i use python=3.8.13 in anaconda, i cannot pip this package
ERROR: Could not find a version that satisfies the requirement pysolidworks (from versions: none)
ERROR: No matching distribution found for pysolidworks

**Comment by sjohnson-FLL** [comment]
I have the same issue. (Python 3.11.)

**Comment by SkyDawgie** [comment]
Also having the same problem. Tried this with Python3.7 & Python3.11

**Comment by adamcodes716** [comment]
I don't believe the errors have anything to do with the version of python.  I was able to get past this by doing something that looks like this:

1.  Clone this repo locally and work from that folder  This should get you past the first error
2.  Create the /pyworks/test folder.  This might not be necessary
4.  pip install pywin32
6.  In setup.py, remove the refrerence to "pythoncom" in "install_requires".  This package is installed as part of pywin32 and is not available as a standalone package.
7. pip install .
